config/routes.rb

```diff
@@ -0,0 +1,7 @@
+Rails.application.routes.draw do
+ get '/api/random_greeting', to: 'api#random_greeting'
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
```

Comment:
- [ ] [REQUIRED] Good job so far! There should be a main static route created:

* The `routes.rb` file has a route created (should be marked as `root` => `root#index`)